sdk/dotnet/Pulumi/Deployment/GrpcEngine.cs

```diff
@@ -16,27 +16,27 @@ internal class GrpcEngine : IEngine
         private readonly Engine.EngineClient _engine;
         // Using a static dictionary to keep track of and re-use gRPC channels
         // According to the docs (https://docs.microsoft.com/en-us/aspnet/core/grpc/performance?view=aspnetcore-6.0#reuse-grpc-channels), creating GrpcChannels is expensive so we keep track of a bunch of them here
-        private static ConcurrentDictionary<string, GrpcChannel> _engineChannels = new ConcurrentDictionary<string, GrpcChannel>();
+        private static readonly ConcurrentDictionary<string, GrpcChannel> _engineChannels = new ConcurrentDictionary<string, GrpcChannel>();
 
         public GrpcEngine(string engineAddress)
         {
             // maxRpcMessageSize raises the gRPC Max Message size from `4194304` (4mb) to `419430400` (400mb)
             var maxRpcMessageSize = 400 * 1024 * 1024;
```

Comment:
I was going to suggest moving this inside the lambda, to avoid the closure allocation, but changing it to `const int` rather than `var` achieves that as well:

```suggestion
            const int maxRpcMessageSize = 400 * 1024 * 1024;
```

Reply:
done ✅